test/build-certificate.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const util = require('util')
+const pem = require('pem')
+
+const createCertificate = util.promisify(pem.createCertificate)
+
+async function buildCertificate () {
+  if (!global.context || !global.context.cert || !global.context.key) {
```

Comment:
I am using `global` instead of `t.context` here because `t.context` is only supported by `t.beforeEach` and `t.afterEach`.
I am using `t.before` inside the test case which can reduce the times of executing this code.

Reply:
Can you add this note to the code?

Reply:
of cuz.